Content.Client/GameObjects/Components/Items/ClientHandsComponent.cs

```diff
@@ -116,7 +116,19 @@ private void _setHand(string hand, IEntity entity)
                 return;
             }
 
-            var item = entity.GetComponent<ItemComponent>();
+            SetInHands(hand, entity);
+        }
+
+        private void SetInHands(string hand, IEntity entity)
+        {
+            if (entity == null)
+            {
+                _sprite.LayerSetVisible($"hand-{hand}", false);
+
+                return;
+            }
+
+            if (!entity.TryGetComponent(out ItemComponent item)) return;
```

Comment:
Out of interest, why is this a safe check now? Should players ever be holding non-items?

Reply:
Yeah. Players are never meant to be holding non-items.

Reply:
It doesn't matter as much because this is merged now - but what I meant was "if players are never meant to be holding non-items, why does this catch that case?". I think code should fail fast if its assumptions don't mesh with reality - so in my mind, this should use `GetComponent`, and not prevent the exception using `TryGetComponent` and then just fail silently.